docs/src/modules/components/AppFrame.js

```diff
@@ -310,7 +310,7 @@ function AppFrame(props) {
               data-ga-event-category="header"
               data-ga-event-action="dark"
             >
-              {theme.palette.type === 'light' ? <Brightness4Icon /> : <Brightness7Icon />}
```

Comment:
That reminds me (and completely incidental to this PR), but we should really provide an option to return to the browser / system setting. A the moment, once a cookie is set, the mode is fixed.